qsiprep/workflows/dwi/pre_hmc.py

```diff
@@ -121,37 +123,37 @@ def init_dwi_pre_hmc_wf(scan_groups,
             else (dwi_series, rpe_series)
         pe_axis = dwi_series_pedir[0]
         plus_source_file = get_source_file(plus_files, suffix='_PEplus')
-        merge_plus = init_merge_and_denoise_wf(
-            raw_dwi_files=plus_files,
-            b0_threshold=b0_threshold,
-            dwi_denoise_window=dwi_denoise_window,
-            unringing_method=unringing_method,
-            dwi_no_biascorr=dwi_no_biascorr,
-            no_b0_harmonization=no_b0_harmonization,
-            denoise_before_combining=denoise_before_combining,
-            orientation=orientation,
-            omp_nthreads=omp_nthreads,
-            source_file=plus_source_file,
-            phase_id=pe_axis + "+ phase-encoding direction",
-            calculate_qc=False,
-            name="merge_plus")
+        merge_plus = init_merge_and_denoise_wf(raw_dwi_files=plus_files,
+                                               b0_threshold=b0_threshold,
+                                               dwi_denoise_window=dwi_denoise_window,
+                                               unringing_method=unringing_method,
+                                               dwi_no_biascorr=dwi_no_biascorr,
+                                               denoise_method=denoise_method,
+                                               no_b0_harmonization=no_b0_harmonization,
+                                               denoise_before_combining=denoise_before_combining,
+                                               orientation=orientation,
+                                               omp_nthreads=omp_nthreads,
+                                               source_file=plus_source_file,
+                                               phase_id=pe_axis + "+ phase-encoding direction",
```

Comment:
Hi @mattcieslak : In your commit you hadn't added this line. I added it to resolve conflicts with `master`. Is that okay?

Reply:
this is ok